python-client/giskard/__init__.py

```diff
@@ -66,81 +57,10 @@ def get_version() -> str:
     'MLFlowBasedModel',
     'CustomModel',
     'Suite',
-    'test_drift_psi',
-    'test_drift_chi_square',
-    'test_drift_ks',
-    'test_drift_earth_movers_distance',
-    'test_drift_prediction_psi',
-    'test_drift_prediction_chi_square',
-    'test_drift_prediction_ks',
-    'test_drift_prediction_earth_movers_distance',
-    'test_right_label',
-    'test_output_in_range',
-    'test_disparate_impact',
-    'AucTest',
-    'test_mae',
-    'test_rmse',
-    'test_recall',
-    'test_auc',
-    'test_accuracy',
-    'test_precision',
-    'test_f1',
-    'test_r2',
-    'test_diff_recall',
-    'test_diff_accuracy',
-    'test_diff_precision',
-    'test_diff_rmse',
-    'test_diff_f1',
-    'test_diff_reference_actual_rmse',
-    'test_diff_reference_actual_accuracy',
-    'test_diff_reference_actual_f1',
-    'test_metamorphic_invariance',
-    'test_metamorphic_increasing',
-    'test_metamorphic_decreasing',
-    'test_metamorphic_decreasing_t_test',
-    'test_metamorphic_increasing_t_test',
-    'test_metamorphic_invariance_t_test',
-    'test_metamorphic_increasing_wilcoxon',
-    'test_metamorphic_decreasing_wilcoxon',
-    'test_metamorphic_invariance_wilcoxon',
```

Comment:
I think it's a good idea, but I don't like the fact that the user has to pull tests now from:
```py
from giskard.ml_worker.testing.tests.X import some_test
```

would it help to only expose at least:
```
giskard.ml_worker.testing.metamorphic
giskard.ml_worker.testing.statistic
giskard.ml_worker.testing.drift
giskard.ml_worker.testing.performance
```
so that a user can do:
```
from giskard.performance import test_f1
```

Reply:
> Another solution is to refactor the tests to a better place from where are right now, on a similar level as `models` and `datasets` as suggested here: #909
> 
> so we'd have something like:
> 
> ```
> from giskard.tests.performance import test_f1
> ```
> 
> or simply:
> 
> ```
> from giskard.tests import test_f1
> ```

yes, for sure, `giskard.tests`  LGTM